jdaviz/core/loaders/importers/spectrum1d/spectrum1d.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <v-container>
+    <plugin-select
+      :items="extension_items.map(i => i.label)"
+      :selected.sync="extension_selected"
+      :show_if_single_entry="true"
+      :multiselect="false"
+      label="Extension"
+      api_hint="ldr.extension ="
+      :api_hints_enabled="api_hints_enabled"
+      hint="Extension to use for the data."
+    ></plugin-select>
+
+    <plugin-auto-label
+      :value.sync="data_label_value"
+      :default="data_label_default"
+      :auto.sync="data_label_auto"
+      :invalid_msg="data_label_invalid_msg"
+      label="Data Label"
+      api_hint="ldr.importer.data_label ="
+      :api_hints_enabled="api_hints_enabled"
+      hint="Label to assign to the new data entry."
+    >
+    </plugin-auto-label>
+
+    <plugin-viewer-create-new
+      :items="viewer_items"
+      :selected.sync="viewer_selected"
+      :create_new_items="viewer_create_new_items"
+      :create_new_selected.sync="viewer_create_new_selected"
+      :new_label_value.sync="viewer_label_value"
+      :new_label_default="viewer_label_default"
+      :new_label_auto.sync="viewer_label_auto"
+      :new_label_invalid_msg="viewer_label_invalid_msg"
+      :multiselect="viewer_multiselect"
+      :show_multiselect_toggle="false"
+      label="Viewer"
+      api_hint='ldr.importer.viewer = '
```

Comment:
Extremely unimportant but double quotes would make this line consistent with the rest. Also space after? Or no space after? The rest have no space after.
```suggestion
      api_hint="ldr.importer.viewer = "
```